src/applications/ivc-champva/10-7959f-1/config/form.js

```diff
@@ -49,6 +53,31 @@ const formConfig = {
   customText: {
     appType: 'form',
   },
+  // This is here temporarily to allow us to log SIP/Prefill data on staging
+  onFormLoaded: props => {
+    // TODO: Remove all this when we've verified we're getting the right data.
+    const { formData, returnUrl } = props;
+    // Check valid return URL; copied from RoutedSavableApp
+    const isValidReturnUrl = checkValidPagePath(
+      props.routes[props.routes.length - 1].pageList,
+      formData,
+      returnUrl,
+    );
+    if (isValidReturnUrl) {
+      props.router.push(returnUrl);
+    } else {
+      const nextPagePath = getNextPagePath(
+        props.routes[props.routes.length - 1].pageList,
+        formData,
+        '/introduction',
+      );
+      props.router.push(nextPagePath);
+    }
+    // Show whatever formData we have at this time, which should include data
+    // produced by the prefill transformer
+    // eslint-disable-next-line no-console
+    console.log('Form loaded - data: ', formData);
+  },
```

Comment:
This function gets the props passed to the form when it is loaded, pulls out the form data, navigates to the first page of the form, and then logs the form data.

That log of the form data is what we're after - we need to see if the prefill transformer is producing the output on staging that we think it ought to, and if not, this will help us fix our prefill transformer so it lines up correctly.